### PR TITLE
testcafe editor's tests: change folder structure 

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: testing/testcafe/tests/editors/label.ts [hunk 1]
```diff
@@ -1,101 +1,30 @@
 import { compareScreenshot } from 'devextreme-screenshot-comparer';
```
Comment (ksercs) [inline testing/testcafe/tests/editors/label.ts]: @groshenkovamarina  please review this changes

File: testing/testcafe/tests/editors/radioGroup/validationMessage.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import CheckBox from '../../../model/checkBox';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`Radio Group ValidationMessage`
+  .page(url(__dirname, '../pages/t1020449.html'));
```
Comment (ksercs) [inline testing/testcafe/tests/editors/radioGroup/validationMessage.ts]: why don't we use `container.html` everywhere?

File: testing/testcafe/tests/editors/textBox/validationMessage.ts [hunk 3]
```diff
@@ -1,34 +1,14 @@
...
+import { restoreBrowserSize } from '../../../helpers/restoreBrowserSize';
+import createWidget from '../../../helpers/createWidget';
+import { appendElementTo } from '../../navigation/helpers/domUtils';
 
 const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 
 fixture`ValidationMessage`
-  .page(url(__dirname, './pages/T941581.html'));
-
-test('ValidationMessage integrated in editor should not raise any errors when it is placed inside of form and has name "style"', async (t) => {
```
Comment (ksercs) [inline testing/testcafe/tests/editors/textBox/validationMessage.ts:14]: are these tests just removed?

File: testing/testcafe/tests/editors/radioGroup/validationMessage.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    .ok()
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+});
+
+fixture`ValidationMessage`
+  .page(url(__dirname, '../../container.html'));
+
+test('ValidationMessage integrated in editor should not raise any errors when it is placed inside of form and has name "style" (T941581)', async (t) => {
+  const checkBox = new CheckBox('#container');
```
Comment (ksercs) [inline testing/testcafe/tests/editors/radioGroup/validationMessage.ts]: shouldn't we use RadioGroup here or move this tests to checkbox package? Now we test CheckBox inside of `radioGroup/validationMessage` package

**Comment by ksercs** [comment]
@EugeniyKiyashko  can we name this pull request more specific? What is a purpose of this changes?

File: testing/testcafe/tests/editors/radioGroup/validationMessage.ts (hunk 2 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/radioGroup/validationMessage.ts]: We may use additional html pages to test scenarios that difficult emulate in test body. Only 2 such test pages left: dateBoxGeometry.html & t901779.html.

File: testing/testcafe/tests/editors/radioGroup/validationMessage.ts (hunk 4 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/radioGroup/validationMessage.ts]: I moved the parts that belong widget into the widget's folder.

File: testing/testcafe/tests/editors/textBox/validationMessage.ts [hunk 5]
```diff
@@ -1,38 +1,18 @@
...
-});
-
-fixture`ValidationMessage_TextBox`
-  .page(url(__dirname, '../container.html'));
+  .page(url(__dirname, '../../container.html'));
 
 test('Validation Message position should be correct after change visibility of parent container (T1095900)', async (t) => {
   const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
-
+  await t.debug();
```
Comment (ksercs) [inline testing/testcafe/tests/editors/textBox/validationMessage.ts]: is it necessary?

**Review by ksercs — DISMISSED** [review]

File: testing/testcafe/tests/editors/textBox/validationMessage.ts (hunk 5 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/textBox/validationMessage.ts]: No, it is not necessary. https://github.com/DevExpress/DevExtreme/pull/22065/commits/58f203c4ac4f006ff41157a9e1542ae663367595

**Review by Zedwag — DISMISSED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**